app/controllers/sales_controller.rb

```diff
@@ -39,6 +39,7 @@ def new
   end
 
   def show
+    @object = @sale
```

Comment:
that makes no sense... the @object here should not be needed... if the sale object without `.decorated` is actually needed, the name of the variable must reflect that, "object" could be anything.

suggestion: `@non_decorated_sale`